content/en/docs/concepts/instrumenting.md

```diff
@@ -1,33 +1,54 @@
 ---
 title: "Instrumenting"
 description: >-
-  How OpenTelemetry facilitates automatic and manual instrumentation of applications
+  How OpenTelemetry facilitates automatic and manual instrumentation of
+  applications
 weight: 40
 ---
 
-In order to make a system observable, it must be instrumented. That is, the code
-must emit [traces](/docs/concepts/observability-primer/#distributed-traces),
+In order to make a system observable, it must be **instrumented**: That is, the
+code must emit
+[traces](/docs/concepts/observability-primer/#distributed-traces),
 [metrics](/docs/concepts/observability-primer/#reliability--metrics), and
-[logs](/docs/concepts/observability-primer/#logs). OpenTelemetry facilitates
-the instrumenting of applications, by providing means to automatically or
-manually instrument your application.
-
-The exact installation mechanism for OpenTelemetry varies based on the language
-you’re developing in, but there are some similarities covered in the sections
- below.
+[logs](/docs/concepts/observability-primer/#logs).
+
+A developer can
+[manually instrument](/docs/reference/specification/glossary/#manual-instrumentation)
+their applications by coding against the OpenTelemetry APIs.
+
+The developer does not need to instrument all the dependencies used in their
+application:
+
+- some of their libraries will be observable out of the box by calling the
+  OpenTelemetry API themselves directly. Those libraries are sometimes called
+  **natively instrumented**.
+- for libraries without such an integration the OpenTelemetry projects provide
+  language specific
+  [Instrumentation Libraries](/docs/reference/specification/overview/#instrumentation-libraries).
+
+To facilitate the instrumentation of applications even more,
+[Automatic Instrumentation](/docs/reference/specification/glossary/#automatic-instrumentation)
+can be used to collect telemetry from an application without the end-user being
+required to modify the application's source code. If you previously used an APM
+agent to automatically extract telemetry from your application, Automatic
+Instrumentation will give you a similar out of the box experience.
```

Comment:
Hmmm, I feel like maybe this should be flipped: start with automatic instrumentation or instrumentation libraries, and then "to facilitate the instrumentation of applications even more," talk about manual instrumentation.

Reply:
but isn't automatic instrumentation the most convenient one? And manual instrumentation is maybe more work but gives you that extra details. Or are you thinking about the flow -> instrument your app automatically -> gradually add manual instrumentation?

Reply:
Yes, the latter, the flow of the page should probably lead with automatric instrumentation (and/or instrumentation libraries), followed by manual instrumentation.

Reply:
fixed, how about now?

Reply:
I like it!